renderer/brave_wallet/DEPS

```diff
@@ -1,5 +1,6 @@
 include_rules = [
   "+brave/common",
+  "+brave/components/version_info",
```

Comment:
this is not needed.

Reply:
yup, going to replace it to `"+components/version_info",`